content/release-notes/2021-04-02.md

```diff
@@ -0,0 +1,27 @@
++++
+date = "2021-04-02T10:00:00-07:00"
+publishdate = "2021-04-02T10:00:00-07:00"
+draft = false
+title = "Release 2021.11"
+images = ["https://crunch.io/img/logo-1200x630.png"]
++++
+
+### Crunch web application
+
+- Updated how descriptions are handled, which now allows them to extend to the entire width beneath a graph as needed.
+- Changed the default to 'aspect ratio' when rendering time plots in "Tables and graphs" mode (in early access).
+- Improved search results by including:
+    - whether or not a dataset is a view, *and*
+    - for variable results, the type is now included.
+- Fixed bugged so that exports of weighted numeric-array analyses are now successful.
+- Improved how stacked vertical bar graphs appear by allocating additional space for vertical axis labels so that they fully display.
+- Improved the time plot feature so that 'time periods' with no data at the beginning or end of a time range are now omitted from the horizontal axis — this is now dynamic as you add or remove categories (in early access).
+
+### Crunch Automation
+
+- Added the new argument `dry` to the Crunch Automation script endpoint, which prevents the script from executing and only performs validations.
```

Comment:
```suggestion
- Added the new argument `dryrun` to the Crunch Automation script endpoint, which prevents the script from executing and only performs validations.
```
if it's not already this, it's not worth a separate entry next week for "changed dry to dryrun"